src/content/docs/tutorial-errors/respond-outages.mdx

```diff
@@ -0,0 +1,155 @@
+
+## Find the critical error [#critical-error]
+
+Now that your apps are instrumented, New Relic is capturing data about your services. This includes data about error occurrences in your app. 
+
+<Steps>
+    <Step>
+        ### Think about the end user
+
+Alerts let you know a problem exists: they're the water on your lawn. But alerts won't provide you all the context. That's where your errors inbox comes in. 
+
+Imagine you're responsible for some apps on an ecommerce site. You've received two alerts for two components, one for checking out and one for searching inventory. You're only receiving reports that the search functionality is failing for end users, but the check out component works fine. You may believe that the check out function is more important, but it's critical to separate your beliefs from your observability practices. 
+
+This practice applies even if the end user hasn't reported anything. When you notice services failing, you can ask yourself these questions:
+
+* Is the end user experiencing a problem? 
+* How should their experience look?
+* What behavior are they currently experiencing?
+
+    </Step>
+    <Step>
+        ### Choose a service
+
+Let's see how this might look in practice. When you view the **All entities** page, you notice four services are alerting.
+
+<img
+title="Overview errors affecting your services"
+alt="A screenshot showing an app with many errors"
+src={allEntitiesView}
+/>
+
+After asking yourself the questions from step one, you know that: 
+
+* The end user is struggling with purchase actions.
+* Your site should only display in-stock items.
+* Your site is displaying all products, so customers are able to purchase out-of-stock items.
+
+After performing your audit, you know that `api-gateway` is a critical dependency for your inventory. This is where you'll begin your error analysis. 
+
+    </Step>
+    <Step>
+
+### Locate what changed [#source] 
+
+You have your entry point into your system, so now you can look into the errors affecting your app. From the `api-gateway` summary page, click the **Errors** tab under **Triage**. Your errors page filters your data to an errors-only view. 
+
+<img
+title="Overview errors affecting your services"
+alt="A screenshot showing an app with many errors"
+src={apmErrorsInboxPage}
+/>
+
+`api-gateway` has at least six error groups reporting with anywhere from a dozen to thousands of occurrences in your app. Don't make any assumptions about what error is critical. Instead, start with the time series on the far right of the waterfall view. 
+
+At first this seems to lack granularity, but your time series gives you enough information to point to what changed over time. We'll break this down:
```

Comment:
What is the `this`? The time series? The waterfall view? The service itself? In general this ¶ is a bit tough for me to connect with the actual product UI.